BUIP-UAHF/buip-uahf-technical-spec.md

```diff
@@ -151,6 +153,9 @@ NOTE 2: The client shall still accept transactions whose signatures
 verify according to pre-fork rules, subject to the additional OP_RETURN
 constraint introduced by REQ-6-1.
 
+NOTE 3: If bit 6 is not set, only the traditional non-shifted nHashType
```

Comment:
are we shifting or adding?
"adding a magic 'fork id' value to the nHashType before the hash is"